kratos/includes/model_part.h

```diff
@@ -11,42 +11,20 @@
 //
 //
 
-
-
 #if !defined(KRATOS_MODEL_PART_H_INCLUDED )
 #define  KRATOS_MODEL_PART_H_INCLUDED
 
-
-
 // System includes
-#include <string>
-#include <iostream>
-#include <sstream>
-#include <cstddef>
-
 
 // External includes
 
-
 // Project includes
-#include "includes/define.h"
 #include "includes/serializer.h"
 #include "includes/process_info.h"
-#include "containers/data_value_container.h"
-#include "includes/mesh.h"
-#include "includes/element.h"
-#include "includes/condition.h"
 #include "includes/communicator.h"
 #include "includes/table.h"
-#include "containers/pointer_vector_map.h"
 #include "containers/pointer_hash_map_set.h"
 #include "input_output/logger.h"
```

Comment:
what abt this?

Reply:
It is already included in other files, don't remember which one